src/TextFieldCatalog.js

```diff
@@ -67,20 +95,24 @@ const Label = ({textFieldId}) => (
   </label>
 );
 
-const Input = ({placeholder, textFieldId, onClick}) => (
+const Input = ({placeholder, textFieldId, onClick, noLabel, ...props}) => (
```

Comment:
What was props added for in these? It's hard for me to tell offhand.

Reply:
Removed the use of `...props` and added `maxLength` attribute explicitly.